homework/index.html

```diff
@@ -18,7 +18,9 @@
   </head>
 
   <body>
-    <div id="root"></div>
+    <header>
```

Comment:
Why wrapping the entire app in `<header>`, it seems only parts of the app should be in header?